pfioh/pfioh.py

```diff
@@ -635,13 +638,13 @@ def ls_process(self, **kwargs):
 
         return d_ret
     
-    def getHeaders(self):
+    def getContentLength(self):
```

Comment:
The name of this function was misleading, so I changed it. It actually gets the headers and returns the value of content-length.